src/analysis/utils/runtime-utils.ts

```diff
@@ -48,6 +48,7 @@ export const defaultLocation = 'US-CENTRAL1';
 
 export const defaultComputeZone = 'US-CENTRAL1-A';
 export const defaultComputeRegion = 'US-CENTRAL1';
+export const defaultAzureComputeRegion = 'eastus';
```

Comment:
I am pretty sure that our default region for Azure is southcentral now, better double check with the dsp-azure channel

Reply:
Yes, was going to say that it is 'southcentral'

Reply:
this may duplicate `defaultAzureRegion` at https://github.com/DataBiosphere/terra-ui/blob/07545d8402606b376ea5605cc01da27faf7711ff/src/libs/azure-utils.js#L10C39-L10C39 - tho that value should probably change to "southcentralus"

Reply:
Hmm... so based on this discussion I'm going to remove putting the default region in this function.

The logic to put it here was initially flawed: if leo isn't returning a region, we should be putting 'unknown' here instead of a point-in-time default value.